adlfs/multithread.py

```diff
@@ -14,24 +14,20 @@
 
 Only implements upload and download of (massive) files and directory trees.
 """
-from concurrent.futures import ThreadPoolExecutor, wait
 import glob
 import logging
-import multiprocessing
 import os
-import pickle
-import time
-import uuid
 
 from .core import AzureDLPath
+from .transfer import ADLTransferClient
 from .utils import commonprefix, datadir, read_block, tokenize
 
 MAXRETRIES = 5
 
 logger = logging.getLogger(__name__)
 
 
-class ADLDownloader:
+class ADLDownloader(object):
```

Comment:
Is this "object" that is passed in documented below so the caller knows what this object should be?